pkg/utils/creds/creds.go

```diff
@@ -12,10 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package e2t
+package creds
 
-import "testing"
+import (
+	"crypto/tls"
 
-func TestDummy(t *testing.T) {
-	t.Log("Yo!")
-}
\ No newline at end of file
+	"github.com/onosproject/onos-lib-go/pkg/certs"
+)
+
+// GetClientCredentials :
+func GetClientCredentials() (*tls.Config, error) {
```

Comment:
I think we should have these stuff in onos-lib-go

Reply:
Agreed. This, and a few other things brought over here, may need to be further relocated for sure.